용재/1935.py

```diff
@@ -0,0 +1,27 @@
+N = int(input())
+getText = input()
+
+inputNumbers = []
+
+for i in range(N):
+  inputNumbers.append(int(input()))
```

Comment:
```suggestion
inputNumbers = [int(input()) for _ in range(N)]
```
이렇게 간단히도 입력받을 수 있을 것 같아요!!

Reply:
저렇게 하면 한층 간결해 지는군요 다음부터 적용해서 풀어보겠습니다. 감사합니다~!